designer/client/i18n/translations/en.translation.json

```diff
@@ -371,5 +371,14 @@
   "titleHint": "Enter the name to show for this component",
   "warning": "Warning",
   "wontShow": "This will not show on the page.",
-  "yes": "yes"
+  "yes": "yes",
+  "saveErrorPage": {
+    "sorry": "Sorry, there is a problem with the service",
+    "error": "An error occurred while saving the component.",
+    "hint": "We saved the last valid version of your form. Return to the Designer to continue.",
```

Comment:
haymon asked me to use Designer

Reply:
ok, I was going for consistency with the usage of "Designer" rather than "designer" in the hyperlink but if he's happy with this then I'll resolve the comment